fractal-puppy/Engine
# Feature/credits video

new credits video added. To test:
- check that from menu you can see the credits
- check that when finishing the game the credits also work
